### ISSUE TITLE
Angulation of penis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10012]
**Preferred term label:**
Angulation of penis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Penis fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000085
- label: Angulation of penis
- parents: [Abnormal penis morphology (HP:0000036)](https://hpo.jax.org/app/browse/term/HP:0000036)
- pmid: PMID:29570261
- def: Abnormally increased curvature of the penis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10012
- synonyms: Bent penis(LAYPERSON_TERM)
